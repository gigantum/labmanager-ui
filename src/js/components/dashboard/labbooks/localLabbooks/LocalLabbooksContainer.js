import Labbooks from '../Labbooks';
import {
  createFragmentContainer,
  graphql,
} from 'react-relay';

export default createFragmentContainer(
  Labbooks,
  graphql`
    fragment LocalLabbooksContainer_labbookList on LabbookQuery{
      labbookList{
        id
        ...LocalLabbooks_localLabbooks
      }
    }
  `,
);
