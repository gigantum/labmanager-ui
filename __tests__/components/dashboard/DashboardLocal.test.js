import {Provider} from 'react-redux'
import React from 'react'
import renderer from 'react-test-renderer';
import history from 'JS/history'
import {mount} from 'enzyme'
import LocalLabbooksContainer from 'Components/dashboard/labbooks/localLabbooks/LocalLabbooksContainer';
import store from "JS/redux/store"
import {BrowserRouter as Router} from 'react-router-dom'

import json from './__relaydata__/DashboardLocal.json'

import relayTestingUtils from 'relay-testing-utils'


const fixtures = {
  auth: ()=>{

  },
  localLabbooks: json.data.labbookList,
  labbookList: json.data.labbookList,
  history: history,
  refetchSort: ()=>{

  }
}

test('Test DashboardLocal snapshot', () => {

  const wrapper = renderer.create(

      relayTestingUtils.relayWrap(
        <Provider store={store}>
          <Router>
            <LocalLabbooksContainer {...fixtures} />
          </Router>
        </Provider>
      , {}, json.data)

  );

  const tree = wrapper.toJSON()

  expect(tree).toMatchSnapshot()

})
