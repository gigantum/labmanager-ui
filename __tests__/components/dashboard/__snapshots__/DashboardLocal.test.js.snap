// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test DashboardLocal snapshot 1`] = `
<div
  className="Labbooks"
>
  <div />
  <div
    className="Labbooks__title-bar"
  >
    <h6
      className="Labbooks__username"
    >
      cbutler
    </h6>
    <h2
      className="Labbooks__title"
      onClick={[Function]}
    >
      Projects
    </h2>
  </div>
  <div
    className="Labbooks__menu  mui-container flex-0-0-auto"
  >
    <ul
      className="Labbooks__nav  flex flex--row"
    >
      <li
        className="Labbooks__nav-item--0 selected"
      >
        <a
          onClick={[Function]}
        >
          Local
        </a>
      </li>
      <li
        className="Labbooks__nav-item--1"
      >
        <a
          onClick={[Function]}
        >
          Cloud
        </a>
      </li>
      <hr
        className="Labbooks__navigation-slider Labbooks__navigation-slider--0"
      />
      <div
        className="hidden cloudLocal"
      >
        <div
          className="ToolTip__toggle cloudLocal"
          onClick={[Function]}
        >
          <div
            className="ToolTip__glow-container"
          >
            <div
              className="ToolTip__glow-ring-outer"
            >
              <div
                className="ToolTip__glow-ring-inner"
              />
            </div>
          </div>
        </div>
        <div
          className="hidden cloudLocal"
        />
        <div
          className="hidden cloudLocal"
        >
          This allows you to switch between Local and Cloud Project listings. Both Projects you have created and Projects that have been shared with you will be visible.
        </div>
      </div>
    </ul>
  </div>
  <div
    className="Labbooks__subheader"
  >
    <div
      className="Labbooks__search-container"
    >
      <input
        className="Labbooks__search no--margin"
        defaultValue=""
        onFocus={[Function]}
        onKeyUp={[Function]}
        placeholder="Filter Projects by name or description"
        type="text"
      />
    </div>
    <div
      className="Labbooks__filter"
    >
      Filter by:
      <span
        className="Labbooks__filter-collapsed"
        onClick={[Function]}
      >
        All
      </span>
      <ul
        className="hidden"
      >
        <li
          className="Labbooks__filter-item"
          onClick={[Function]}
        >
          All 
          ✓ 
        </li>
        <li
          className="Labbooks__filter-item"
          onClick={[Function]}
        >
          My Labbooks 
          
        </li>
        <li
          className="Labbooks__filter-item"
          onClick={[Function]}
        >
          Shared with me 
          
        </li>
      </ul>
    </div>
    <div
      className="Labbooks__sort"
    >
      Sort by:
      <span
        className="Labbooks__sort-collapsed"
        onClick={[Function]}
      >
        Modified Date (Newest)
      </span>
      <ul
        className="hidden"
      >
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Modified Date (Newest) 
          ✓ 
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Modified Date (Oldest) 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Creation Date (Newest) 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Creation Date (Oldest) 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          A-Z 
          
        </li>
        <li
          className="Labbooks__sort-item"
          onClick={[Function]}
        >
          Z-A 
          
        </li>
      </ul>
    </div>
  </div>
  <div
    className="LocalLabbooks__labbooks"
  >
    <div
      className="LocalLabbooks__sizer grid"
    >
      <div
        className="ImportModule Labbooks__panel Labbooks__panel--add Labbooks__panel--import column-4-span-3"
        id="dropZone"
        onDragEnd={[Function]}
        onDragOver={[Function]}
        onDrop={[Function]}
        type="file"
      >
        <div
          className="Labbooks__labbook-main"
        >
          <div
            className="Labbooks__labbook-header"
          >
            <div
              className="Labbooks__labbook-icon"
            >
              <div
                className="Labbooks__title-add"
              />
            </div>
            <div
              className="Labbooks__add-text"
            >
              <h4>
                Add Project
              </h4>
            </div>
          </div>
          <div
            className="Labbooks__labbook-button"
            onClick={[Function]}
          >
            Create New
          </div>
          <div
            className="hidden createLabbook"
          >
            <div
              className="ToolTip__toggle createLabbook"
              onClick={[Function]}
            >
              <div
                className="ToolTip__glow-container"
              >
                <div
                  className="ToolTip__glow-ring-outer"
                >
                  <div
                    className="ToolTip__glow-ring-inner"
                  />
                </div>
              </div>
            </div>
            <div
              className="hidden createLabbook"
            />
            <div
              className="hidden createLabbook"
            >
              Clicking on the 'Create New' button will open the new Project window, letting you name and select a Base for the new Project.
            </div>
          </div>
          <div
            className="Labbooks__labbook-button-import"
            onClick={[Function]}
          >
            Import Existing
          </div>
          <div
            className="hidden importLabbook"
          >
            <div
              className="ToolTip__toggle importLabbook"
              onClick={[Function]}
            >
              <div
                className="ToolTip__glow-container"
              >
                <div
                  className="ToolTip__glow-ring-outer"
                >
                  <div
                    className="ToolTip__glow-ring-inner"
                  />
                </div>
              </div>
            </div>
            <div
              className="hidden importLabbook"
            />
            <div
              className="hidden importLabbook"
            >
              Clicking on the Import Existing' button transition the card to Import mode. 
               You can import by using a URL of a published Project or by uploading an exported Project's .zip file. 
               You can also directly import Projects that have been shared with you in the Cloud view.
            </div>
          </div>
        </div>
        <div
          className="hidden"
        />
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  857255a0-f22e-46b5-b70f-95af00d5d17e
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                This is a test labbook
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  random-picture-generator
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                This labBook create random piuctures.
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  trump-tweets
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by dmk
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                A LabBook to analyze Trumps stupid tweets
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  awful-intersections-demo
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by rjvgigantum
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                A fun demo LabBook to automatically find and generate cringe-worthy maps of horrible intersections
                
                
                
                
                
                
                Some of the code in this LabBook was originally created by Jordan Matelsky (@j6m8) and was inspired by amazing work by Peter Gorman at BarelyMaps
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  labbook-test-activity
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  data-shader
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  917e4606-8d98-46cb-986c-4dd424e0bff7
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                This is a test labbook
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  11c2d7ed-f4ea-4e43-ae77-cf1b7388ea0d
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                This is a test labbook
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  new-labbook-345
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  log-volume-audit
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by bill2
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                An analysis for figuring out where most of the logging material comes from, and to identifiy places to eliminated it.
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  does-this-fail-on-syncing
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by bill2
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                Check if this fails on sync
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  asdasasd
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  temp-labook22
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                this is a description of a labbook sadodaadadad adsdasadsdassad ssaddadsads adsadsdad adsd adasdsadsjkdsjfjnfjdjfdjnfdjfdjfdjn . asdad das ad sad sd asd dsa d ads adsad sad . da das sad das da d da sd da dada ad s d ad
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  asdaaddsad
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  test-lb
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by dmk
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                This a short description
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  labbook-37
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  labbook-38
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  testing-activity-feed
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                testing labbook activity feed
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  test-activity
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            />
          </p>
        </div>
      </div>
      <div
        className="LocalLabbooks__panel column-4-span-3 flex flex--column justify--space-between"
        onClick={[Function]}
      >
        <div
          className="LocalLabbooks__icon-row"
        >
          <div
            className="LocalLabbooks__containerStatus"
          >
            <button
              className="LocalLabbooks__containerStatus--state loading"
              onClick={[Function]}
              onMouseOut={[Function]}
              onMouseOver={[Function]}
            >
              
            </button>
          </div>
        </div>
        <div
          className="LocalLabbooks__text-row"
        >
          <div
            className="LocalLabbooks__title-row"
          >
            <h6
              className="LocalLabbooks__panel-title"
              onClick={[Function]}
            >
              <span
                className={undefined}
              >
                <span
                  className=""
                  style={undefined}
                >
                  ec6d81ca-8c4b-4afc-92ff-9b4eed018156
                </span>
              </span>
            </h6>
          </div>
          <p
            className="LocalLabbooks__owner"
          >
            Created by cbutler
          </p>
          <p
            className="LocalLabbooks__description"
          >
            <span
              className={undefined}
            >
              <span
                className=""
                style={undefined}
              >
                This is a test labbook
              </span>
            </span>
          </p>
        </div>
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
      <div
        className="Labbook__loader-hidden"
      >
        <div />
      </div>
    </div>
  </div>
</div>
`;
